nupic/regions/PCANode.py

```diff
@@ -435,36 +435,3 @@ def getOutputElementCount(self, name):
     else:
       raise Exception('Unknown output: ' + name)
 
-
-
-if __name__=='__main__':
-
-  from nupic.engine import *
-
-  rgen = numpy.random.RandomState(37)
-
-  inputSize = 8
-
-  net = Network()
-  sensor = net.addRegion('sensor', 'py.ImageSensor' ,
-          '{ width: %d, height: %d }' % (inputSize, inputSize))
```

Comment:
added as a test from PCANode in nupic, without the `ImageSensor` stuff which is from nupic.vision; 
in the last commit, but I haven't tested it at all, as I dont have a running nupic now :sob: